draft-ietf-rats-architecture.md

```diff
@@ -1250,60 +1251,55 @@ an Attestation Result for any other purpose.
-In this case, confidentiality of the generator, as well as the path over which the key is provisioned, is necessary.
-The manufacturer needs to take care to protect it with measures consistent with its value.
-This can be achieved in a number of ways.
+One way to provision key material is to first generate it external to the device and then copy the key onto the device.
+In this case, confidentiality protection of the generator, as well as for the path over which the key is provisioned by, is necessary.
+The manufacturer needs to take care to protect corresponding key material with measures appropriate for its value.
 
-Confidentiality can be achieved entirely with physical provisioning facility security involving no encryption at all. For low-security use cases, this might be simply locking doors and limiting personnel that can enter the facility. For high-security use cases, this might involve a special area of the facility accessible only to select security-trained personnel.
+Confidentiality protection can be realized via physical provisioning facility security involving no encryption at all. For low-security use cases, this might be simply locking doors and limiting personnel that can enter the facility. For high-security use cases, this might involve a special area of the facility accessible only to select security-trained personnel.
 
-Cryptography can also be used to support confidentiality, but keys that are used to then provision attestation keys must somehow have been provisioned securely beforehand (a recursive problem).
+Typically, cryptography is used to enable confidentiality protection. This can result in recursive problems, as the key material used to provision attestation keys must again somehow have been provisioned securely beforehand (requiring an additional level of protection, and so on).
 
-In many cases both some physical security and some cryptography will be necessary and useful to establish confidentiality.
+In general, a combination of some physical security measures and some cryptographic measures is used to establish confidentiality protection.
 
-Another way to provision the key material is to generate it on the device and export the verification key. If public key cryptography is being used, then only integrity is necessary. Confidentiality is not necessary.
+Another way to provision key material is to generate it on the device and export the validation key. If public-key cryptography is being used, then only integrity is necessary. Confidentiality is not necessary.
 
-In all cases, the Attestation Key provisioning process must ensure that only attestation key material that is generated by a valid Endorser is established in Attesters and then configured correctly.
-For many use cases, this will involve physical security at the facility, to prevent unauthorized devices from being manufactured that may be counterfeit or incorrectly configured.
```

Comment:
Why was this sentence removed?   I don't think the point about counterfeit devices appears elsewhere does it?